lib/ZumoOta/src/FlashManager.h

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+    DESCRIPTION
+*******************************************************************************/
+/**
+ * @brief  ZumoOta application
+ * @author Decareme Pauline Ngangnou <ngandeca@yahoo.fr>
+ * @{
+ */
+#ifndef FlashManager_H
+#define FlashManager_H
+
+/******************************************************************************
+ * Includes
+ *****************************************************************************/
+#include <ESPAsyncWebServer.h>
+/******************************************************************************
+ * Macros
+ *****************************************************************************/
+ 
+/******************************************************************************
+ * Types and Classes
+ *****************************************************************************/
+
+class FlashManager{
+private:
+        
+public:
+    /**
+     * Constructor of  MySettings.

```

Comment:
MySettings ??

Reply:
So far, Gabryel's settings have not been taken into account.We plan to do it in the next commit.